### PR TITLE
Bold most applicable choice in combat

### LINKED ISSUE #254: Bold most applicable choice in combat
<a href="https://github.com/toddmedema"></img></a> **Issue by [toddmedema](https://github.com/toddmedema)**
_Thursday May 17, 2018 at 13:05 GMT_
_Originally opened as https://github.com/ExpeditionRPG/expedition-app/issues/690 (0 comment(s))_

----

This will involve some design experimentation to find a subtle but visible way to "highlight" buttons on the black combat cards. Bolding alone didn't seem to be visible enough (especially on a phone screen), so perhaps bolding + a slightly thicker border and slightly lighter background. Make sure to include screenshots both in desktop view and of it running in the Chrome mobile emulator!

The "most applicable choice" should in general be next, unless: if tier = 0, only bold victory, else if adventurers = 0, only bold defeat (note that if both = 0, only victory should be bolded).

### PR DESCRIPTION
**Pull request by smartin015**
image

Fixes #254

### CODE REVIEW

File: services/app/src/components/views/quest/cardtemplates/combat/PlayerTier.tsx [hunk 1]
```diff
@@ -72,9 +72,9 @@ export default function playerTier(props: Props): JSX.Element {
...
       </Picker>
       {helpText}
-      <Button onClick={() => (shouldRunDecision) ? props.onDecisionSetup(props.node, props.seed) : props.onNext(nextCard)} disabled={props.numAliveAdventurers <= 0}>Next</Button>
-      <Button onClick={() => props.onVictory(props.node, props.settings, props.maxTier, props.seed)}>Victory (Tier = 0)</Button>
-      <Button onClick={() => props.onDefeat(props.node, props.settings, props.maxTier, props.seed)}>Defeat (Adventurers = 0)</Button>
+      <Button className={(props.numAliveAdventurers === 0 || props.tier === 0) ? 'subtle' : ''} onClick={() => (shouldRunDecision) ? props.onDecisionSetup(props.node, props.seed) : props.onNext(nextCard)} disabled={props.numAliveAdventurers <= 0 || props.tier <= 0}>Next</Button>
+      <Button className={(props.tier !== 0) ? 'subtle' : ''} onClick={() => props.onVictory(props.node, props.settings, props.maxTier, props.seed)}>Victory (Tier = 0)</Button>
+      <Button className={(props.numAliveAdventurers !== 0) ? 'subtle' : ''} onClick={() => props.onDefeat(props.node, props.settings, props.maxTier, props.seed)}>Defeat (Adventurers = 0)</Button>
```
Comment (toddmedema) [inline services/app/src/components/views/quest/cardtemplates/combat/PlayerTier.tsx]: The tricky edge case: this should also be subtle if adventurers = 0 AND enemies = 0 - that's actually where this ticket originated from, users were unclear on if they win or not if they kill all enemies and die in the same turn (answer, they win not lose)

File: services/app/src/components/views/quest/cardtemplates/combat/PlayerTier.tsx [hunk 2]
```diff
@@ -72,9 +72,9 @@ export default function playerTier(props: Props): JSX.Element {
         {props.settings.showHelp && <span>The number of adventurers &gt; 0 health.</span>}
       </Picker>
       {helpText}
-      <Button onClick={() => (shouldRunDecision) ? props.onDecisionSetup(props.node, props.seed) : props.onNext(nextCard)} disabled={props.numAliveAdventurers <= 0}>Next</Button>
-      <Button onClick={() => props.onVictory(props.node, props.settings, props.maxTier, props.seed)}>Victory (Tier = 0)</Button>
-      <Button onClick={() => props.onDefeat(props.node, props.settings, props.maxTier, props.seed)}>Defeat (Adventurers = 0)</Button>
+      <Button className={(props.numAliveAdventurers === 0 || props.tier === 0) ? 'subtle' : ''} onClick={() => (shouldRunDecision) ? props.onDecisionSetup(props.node, props.seed) : props.onNext(nextCard)} disabled={props.numAliveAdventurers <= 0 || props.tier <= 0}>Next</Button>
```
Comment (toddmedema) [inline services/app/src/components/views/quest/cardtemplates/combat/PlayerTier.tsx]: Should we perhaps disable victory and defeat in nonsense cases as well, just for a bit more protection against fat fingers? IE disable victory if they've clearly lost (adventurers = 0) and defeat if they've won (tier = 0)?

File: services/app/src/components/views/quest/cardtemplates/combat/PlayerTier.tsx (hunk 1 above)
Comment (smartin015) [reply services/app/src/components/views/quest/cardtemplates/combat/PlayerTier.tsx]: done

File: services/app/src/components/views/quest/cardtemplates/combat/PlayerTier.tsx (hunk 2 above)
Comment (smartin015) [reply services/app/src/components/views/quest/cardtemplates/combat/PlayerTier.tsx]: done

**Pull request merged**